### DISCUSSION TITLE
Schema validation with custom enum and mysql

### DISCUSSION THREAD

**Discussion by pvrdatadome** [prisma/prisma#23240] [Q&A]
### Bug description

Hi team,

I have an issue after pulling tables from the database. I've multiple enums that are not validate; for example:
```
enum test_forbidden_code {
  // 403 @map("403")
  // 401 @map("401")
}
```

So when I'm trying to generate the client I get an error validation message "This line is not an enum value definition". Is there a way to fix/bypass that? valid

### How to reproduce

* having a custom enum in MySQL db as shown as the previous block

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: macOS
- Database: MySQL
- Node.js version: v18.12.1

### Prisma Version

```
prisma                  : 5.10.1
@prisma/client          : 5.10.1
Computed binaryTarget   : darwin-arm64
Operating System        : darwin
Architecture            : arm64
Node.js                 : v18.12.1
Query Engine (Node-API) : libquery-engine 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.10.0-34.5a9203d0590c951969e85a7d07215503f4672eb9
Default Engines Hash    : 5a9203d0590c951969e85a7d07215503f4672eb9
Studio                  : 0.498.0
```

**Comment by janpio** [comment]
Prisma does not know how to fix these names in a way that it automatically becomes a valid enum. Wecan not use numbers (and other special characters) as enum values right now, but we can also not fix this automatically - so we decided to let the user fix this. Affer you remove the `//` and replace the invalid name with e.g. `fourhundredthree` any further `db pull` executions will keep that custom name for this enum value. You just need to fix it once.

Does that help?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
